mne_bids/datasets.py

```diff
@@ -10,23 +10,37 @@
 import os.path as op
 import shutil
 import tarfile
+import requests
 
 from mne.utils import _fetch_file
 
 
-def fetch_faces_data(data_path, repo, subject_ids):
+def fetch_faces_data(data_path=None, repo='ds000117', subject_ids=[]):
     """Dataset fetcher for OpenfMRI dataset ds000117.
 
     Parameters
     ----------
-    data_path : str
-        Path to the folder where data is stored
+    data_path : str | None
+        Path to the folder where data is stored. Defaults to
+        '~/mne_data/mne_bids_examples'
     repo : str
-        The folder name (typically 'ds000117')
+        The folder name. Defaults to 'ds000117'.
     subject_ids : list of int
-        The subjects to fetch
+        The subjects to fetch. Defaults to an empty list, which will
+        not download anything.
```

Comment:
Why so? The function will not do what it is supposed to do ...

Reply:
would changing it to `[1]` be fine with you? Concerning the "why so" --> this function didn't have defaults before and I did not want to impose defaults that are based on my personal arbitrary decisions, so I tried to stay neutral